default.nix

```diff
@@ -27,11 +27,12 @@ let
         "scripts/.*"
         "secrets/.*"
         "specs/.*"
+        ".*\\.nix"
```

Comment:
This should happen automatically, since these are not whitelisted

Reply:
I'll check, but I think I added it because I had rebuilds on *.nix changes.

Reply:
Nah, I removed it, immediately caused a rebuild.